package.json

```diff
@@ -4,12 +4,15 @@
   "description": "Fast and low overhead web framework called fastify benchmarks.",
   "main": "index.js",
   "scripts": {
-    "start": "node benchmark.js"
+    "start": "node benchmark.js",
+    "compare": "node benchmark.js compare",
+    "test": "standard | snazzy",
+    "standard": "standard | snazzy"
   },
   "bin": {
     "benchmark": "./benchmark.js"
   },
-  "repository": "https://github.com/cagataycali/fastify-benchmarks",
+  "repository": "https://github.com/fastify/benchmarks.git",
```

Comment:
It should be without `.git`

Reply:
![repo-ext](https://user-images.githubusercontent.com/321201/30778180-ead0eeca-a09c-11e7-9890-2afabf021b3c.gif)

Reply:
You're right but npm add link without .git that's the reason I said this :)

Reply:
Okay.